doc/en/customize.rst

```diff
@@ -15,17 +15,31 @@ which were registered by installed plugins.
 .. _rootdir:
 .. _inifiles:
 
-initialization: determining rootdir and inifile
+Initialization: determining rootdir and inifile
 -----------------------------------------------
 
 .. versionadded:: 2.7
 
-pytest determines a "rootdir" for each test run which depends on
+pytest determines a ``rootdir`` for each test run which depends on
 the command line arguments (specified test files, paths) and on
-the existence of inifiles.  The determined rootdir and ini-file are
-printed as part of the pytest header.  The rootdir is used for constructing
-"nodeids" during collection and may also be used by plugins to store
-project/testrun-specific information.
+the existence of ``ini-files``.  The determined ``rootdir`` and ``ini-file`` are
+printed as part of the pytest header during startup.
+
+Here's a summary what ``pytest`` uses ``rootdir`` for:
+
+* Constructing ``nodeids`` during collection; each test is assigned
+  a unique ``nodeid`` which is rooted at the ``rootdir`` and takes in account full path,
```

Comment:
nitpick: Why would `nodeid` and `ini-file` (both singular/plural) be monospaced? Those are "concepts", not variable names or code, right?